api/controllers/v0.3/endpoints.js

```diff
@@ -127,19 +128,11 @@ async function setupEndpoint(endpointFn, req, res) {
     soLicenseKey =
       soLicenseKey || (await awsDecrypt.decryptKMS(process.env.SO_LICENSE_KEY));
   } catch (error) {
-    const localErrorMessage =
-      "The ILS client key and/or secret, or the Service Objects license key were not decrypted";
-
-    const errorResponseData = modelResponse.errorResponseData(
```

Comment:
This was mostly redundant and now I think using just one function, `collectErrorResponseData` with the error object, is easier to understand. It just convert the error object and also logs it.